db/schema_generator/htmlGenerator.js

```diff
@@ -32,11 +28,7 @@ var textBlockGenerator = (description) => {
     var details = headerAndLisGenerator('Details');
 
     textBlock = 
-    `<h1>${title}</h1>
-     <p>${description}</p>
-     ${benefits}
-     ${details}
-     `;
+    `<h1>${title}</h1><p>${description}</p>${benefits}${details}`;
 
```

Comment:
This is an interesting way of putting info in the DB. It makes sense for this case, but in real world situations it would make the info hard to use in other ways. It would basically mean more columns in your DB and more code in react. It probably doesn't make a ton of sense to re-do it for this project, just a thought going forward.

Reply:
yeah, this was the part that damien advised to make it all into an html string versus having a table for benefits and a table for details. both being a one to many relationship with shoes. that's how i had it at first, tables for the info. that way it could be extracted and used in different situations versus being hardcoded like you're saying.

and then leslie mentioned putting html strings into the db.

so i ended up going with the html strings.

so yeah, it definitely was weird for me at first. so i actually ended up redoing the approach you're talking about.  i erased my tables, and made the html generator.